docs/design/StratisSoftwareDesign.lyx

```diff
@@ -4408,5 +4416,396 @@ Journald interactions
 Nfs interactions
 \end_layout
 
+\begin_layout Part*
```

Comment:
No, we want it in a separate file and included into the doc. I believe I figure out how to do that once...